.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: run-tests
+
+on: [push, pull_request]
+
+jobs:
+  run-tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-18.04
+          - ubuntu-latest
+        vim-flavour:
+          - vim
+          - neovim
+        vim-from-testing:
+          - true
+          - false
+        exclude:
+          - os: ubuntu-18.04
+            vim-from-testing: true
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Add hirsute repos
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.vim-from-testing }}
+        run: sudo apt-add-repository 'deb http://archive.ubuntu.com/ubuntu/ impish main universe'
```

Comment:
It says "hirsute" above but "impish" here. Is that intentional?

Reply:
I don't think the bleeding edge will be added to Github Actions as an out-of-the-box option, so I'll put a simpler 'testing' here.

The idea is that by only changing the distribution name one can bump the testing.